src/analysis/max_min_driving_force.jl

```diff
@@ -126,13 +165,20 @@ function max_min_driving_force(
         mod(model, opt_model)
```

Comment:
I recall we already discussed once but why run modifications _before_ setting objective? This way the user can't have a look at the actual objective and try to break it (e.g. add some extra small goal to it)

Reply:
put it after now :)